source/crud/tests/v1/write/deleteMany-collation.json

```diff
@@ -14,6 +14,7 @@
     }
   ],
   "minServerVersion": "3.4",
+  "serverless": "forbid",
```

Comment:
This test didn't fail in C# driver either, because the `deleteMany` is executed as `bulkWrite` operation with a single `deleteMany` request.